### PR TITLE
JBPM-5965: Creating wrong multiple fields when dropping unbinded fields.

### PR DESCRIPTION
**Pull request by pefernan**
This fixes the issue about adding more than one field from the components palette.

BTW it also fixes an issue on FieldPropertiesEditor where the field type selector wasn't displaying the right field type.

### CODE REVIEW

**Comment by pefernan** [comment]
@jsoltes can you take a look?

**Comment by jsoltes** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
@pefernan IDK if you're affected by the REST failing tests (see my email to BSIG..)

**Comment by pefernan** [comment]
@manstis yes I am... I saw your message :S

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
jenkins retest this

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
I hate you jenkins

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

File: kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/test/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorHelperTest.java [hunk 1]
```diff
@@ -31,81 +31,80 @@
...
-import org.uberfire.mocks.EventSourceMock;
 
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
 
 @RunWith(GwtMockitoTestRunner.class)
 public class FormEditorHelperTest {
 
-    private List<FieldDefinition> employeeFields;
+    List<FieldDefinition> employeeFields;
```
Comment (hasys) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/test/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorHelperTest.java]: I don't see any reasons to make fields package protected fields. @pefernan can I ask you to return it back?

File: kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/test/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorHelperTest.java [hunk 2]
```diff
@@ -31,81 +31,80 @@
...
+    FormModelerContent content;
 
     @Before
     public void setUp() throws Exception {
         initFields();
         loadContent();
     }
 
-    protected void loadContent() {
+    void loadContent() {
```
Comment (hasys) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/test/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorHelperTest.java]: The same for methods.

File: kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/test/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorHelperTest.java [hunk 3]
```diff
@@ -273,8 +280,8 @@ public void testRemoveFieldsAddToAvailableNoFields() {
                          false);
     }
 
-    protected void testRemoveFields(boolean addToAvailable,
-                                    boolean definitionHasFields) {
+    void testRemoveFields(boolean addToAvailable,
```
Comment (hasys) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-client/src/test/java/org/kie/workbench/common/forms/editor/client/editor/FormEditorHelperTest.java]: And for new methods.

**Comment by pefernan** [comment]
@hasys better now?

**Comment by hasys** [comment]
Hi @pefernan,
yes, it's better now, thank you!

**Pull request merged**